packages/@aws-cdk/aws-redshift/lib/database-query.ts

```diff
@@ -0,0 +1,103 @@
+import * as path from 'path';
```

Comment:
This file looks to be not exported. Stick it in a folder named `private`, OR start the file name with an `_` to make that very clear to the reader.